app/models/pawn.rb

```diff
@@ -1 +1,52 @@
-class Pawn < Piece; end
+# may only move one square forward unless first move
+# if first move, can choose to move one square or two
+# can never move backwards
+# cannot move forward if square in front occupied by another piece
+# captures by taking one square diagonally forward
+
+class Pawn < Piece
+  def valid_move?(destination_x, destination_y)
+    return false if current_position?(destination_x, destination_y)
+    return false if backwards_move?(destination_y)
+    return false if horizontal_move?(destination_x, destination_y)
+    return true if capture_move?(destination_x, destination_y)
+    allowed_to_move?(destination_x, destination_y) && !square_occupied?(destination_x, destination_y)
+  end
+
+  def capture_move?(destination_x, destination_y)
+    x_difference = (destination_x - x_position).abs
+    y_difference = (destination_y - y_position).abs
+    capture_piece = Piece.exists?(x_position: destination_x, y_position: destination_y, is_white: !is_white, game: game)
+    return true if capture_piece && x_difference == 1 && y_difference == 1
+  end

```

Comment:
Last line could be `capture_piece && x_difference == 1 && y_difference == 1`

Without the `return true`

In Ruby, all methods automatically return whatever the last line is. 😄